scripts/docker/Dockerfile.nvcc

```diff
@@ -36,19 +41,28 @@ RUN cd /tmp && \
     cd / && \
     rm -rf /tmp/hdf5-1.12.2*
 
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+RUN mkdir /tmp/build-adios2 && cd /tmp/build-adios2 && \
+    git clone https://github.com/ornladios/ADIOS2.git ADIOS2 && \
+    mkdir adios2-build && cd adios2-build && \
+    cmake ../ADIOS2 -DADIOS2_USE_Blosc2=ON -DADIOS2_USE_Fortran=OFF && \
+    make -j8 && make install && \
+    cd / && \
+    rm -rf /tmp/build-adios2
 
-RUN curl -L https://github.com/Kitware/CMake/releases/download/v3.23.2/cmake-3.23.2-linux-x86_64.tar.gz -o cmake-3.23.2-linux-x86_64.tar.gz && \
-    tar -xzf cmake-3.23.2-linux-x86_64.tar.gz -C /opt
+RUN mkdir /tmp/build-openpmd && cd /tmp/build-openpmd && \
+    git clone https://github.com/openPMD/openPMD-api.git && \
+    mkdir openPMD-api-build && cd openPMD-api-build && \
+    cmake ../openPMD-api -DopenPMD_USE_PYTHON=ON -DPython_EXECUTABLE=$(which python3) -DopenPMD_USE_ADIOS2=ON && \
+    cmake --build . && \
+    cmake --build . --target install && \
+    cd / && \
+    rm -rf /tmp/build-openpmd
 
-ENV PATH=/opt/cmake-3.23.2-linux-x86_64/bin:$PATH
+COPY ascent_build /tmp/ascent_build
 
-COPY build_ascent_cuda.sh /tmp/build-ascent/build_ascent_cuda.sh
+## NOTE: with enable_cuda=ON, arm64 Dockerfile builds fail for some reason
 
-RUN cd /tmp/build-ascent && \
-    bash build_ascent_cuda.sh && \
+RUN cd /tmp/ascent_build && \
+    env enable_cuda=ON bash build_ascent.sh && \
```

Comment:
This works for me when `enable_cuda=OFF`, but currently fails (for unknown reasons) when it is `ON`.